src/modules/72-templates-library/components/TemplateActivityLog/InfiniteScroll.tsx

```diff
@@ -77,7 +80,13 @@ export const useInfiniteScroll = (props: InfiniteScrollProps): InfiniteScrollRet
         hasMore.current = canFetchMore
 
         const responseContent = response.data.content
-        setItems((prevItems: any) => [...prevItems, ...responseContent])
+        setItems((prevItems: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9264786)
```suggestion
        setItems((prevItems: unknown) => {
```